frontend/packages/console-dynamic-plugin-sdk/src/api/api-mocks.ts

```diff
@@ -0,0 +1,19 @@
+export const MockImpl = () => {
+  throw new Error('You need to configure webpack externals to use this function at runtime.');
+};
+
+export const MockCoFetchJSONImpl = () => {
+  throw new Error('You need to configure webpack externals to use this function at runtime.');
+};
+MockCoFetchJSONImpl.delete = () => {
```

Comment:
This code should be co-located with associated API type export, see my other comment below.

Reply:
Done.